### DISCUSSION TITLE
When attempting to execute the profiling task, an error is encountered: 'program go_tls_read_ret: load program: permission denied'.

### DISCUSSION THREAD

**Discussion by Mr-XGJ** [apache/skywalking#12234] [Q&A]
executing profiling task failure, taskId: c2b3dc9fa9ea499cbca05b4d7899b93dc6941893b0129cf207d62d35a89664d9, reason: field GoTlsReadRet: program go_tls_read_ret: load program: permission denied: 632: (15) if r9 == 0x0 goto pc+2: R0=map_value(id=0,off=0,ks=4,vs=36,imm=0) R6=invP(id=0,umax_value=4294967295,var_off=(0x0; 0xffffffff)) R7=map_value(id=0,off=0,ks=8,vs=144,imm=0) R8=invP(id=0) R9=map_value(id=0,off=0,ks=4,vs=36,imm=0) R10=fp0 fp-8=mmmmmmmm fp-16=mmmmmmmm fp-24=mmmmmmmm fp-32=mmmmmmmm fp-40=mmmmmmmm fp-48=mmmmmmmm fp-56=mmmmmmmm fp-64=mmmmmmmm fp-72=mmmmmmmm fp-80=map_value fp-88=mmmmmmmm (truncated, 822 line(s) omitted)" module=profiling.task

**Comment by wu-sheng** [comment]
@mrproliu Any idea?

**Reply by Mr-XGJ** [reply]
Linux CentOS environment, having deployed Skywalking-Rover using the root, when trying to run a profiling task, I face an error stating 'permission denied' Could you advise on the possible reasons for this issue?
